web-marketplace/src/pages/ProjectCreate/hooks/useProjectSaveAndExit.ts

```diff
@@ -0,0 +1,19 @@
+import { useNavigate } from 'react-router-dom';
+
+import { useCreateProjectContext } from '../ProjectCreate';
+
+export const useProjectSaveAndExit = () => {
+  const { formRef, shouldNavigateRef } = useCreateProjectContext();
+  const navigate = useNavigate();
+
+  const saveAndExit = async (): Promise<void> => {
+    shouldNavigateRef.current = false;
+    await formRef.current?.submitForm();
```

Comment:
This is already done in all submit calls.

Reply:
I was testing the following: 
- Edit an existing off-chain project (name for instance)
- Save & exit
- on the projects list, the name isn't updated

But testing this further, it seems to be an issue even when hitting save & next so I guess this could be handled in a separate issue.

Reply:
I think I better solve this as part of my existing PR https://github.com/regen-network/regen-web/pull/2060, since it's changing how we query projects data on the user projects list